tests/c_api_tests/test_c_api_meta_data.cpp

```diff
@@ -2,12 +2,13 @@
 //
 // SPDX-License-Identifier: MPL-2.0
 
-#include <memory>
-
 #include "c_api_cpp_handle.hpp"
 #include "doctest/doctest.h"
```

Comment:
`doctest` is an external library and should be included `#include <doctest/doctest.h>`